src/app/lib/main.dart

```diff
@@ -17,27 +18,40 @@ import 'package:app/business/environment/environment_manager.dart';
 import 'package:app/business/forced_update/update_required_service.dart';
 import 'package:app/business/kill_switch/kill_switch_service.dart';
 import 'package:app/firebase_options.dart';
+import 'package:app/business/logger/logger_manager.dart';
 import 'package:dio/dio.dart';
 import 'package:firebase_core/firebase_core.dart';
 import 'package:firebase_remote_config/firebase_remote_config.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_dotenv/flutter_dotenv.dart';
 import 'package:get_it/get_it.dart';
+import 'package:logger/logger.dart';
+
+late Logger _logger;
 
 Future<void> main() async {
   WidgetsFlutterBinding.ensureInitialized();
   await _registerAndLoadEnvironment();
+  await _registerAndLoadLoggers();
+
+  _logger.d("Initialized environment and logger. Initializing services.");
```

Comment:
Is it normal to have 'Initializing services.' two times here?

Reply:
hum yeah that wasn't my intentions haha. I wanted to mentions that Initialize all services, but maybe I can reword it differently

Reply:
I redo the logs so it makes more sense